apps/web/src/components/AssessmentResultsTable/utils.tsx

```diff
@@ -54,9 +56,12 @@ export const columnHeader = (header: string, status: ColumnStatus) => {
     >
       {Icon && (
         <Icon
-          {...iconColorMap[status.color ?? "gray"]}
-          data-h2-height="base(x1)"
+          role="img"
+          title={status.ariaLabel}
+          aria-hidden="false"
```

Comment:
Title attribute is deprecated, can we use `aria-label` instead? Also, I believe with `aria-hidden="false" the `role="img"` is implicit so if you want you should be able to remove that as well.

Reply:
I used that initially, but kept finding suggestions to use the role="img" with a `<title />` (which adds a nice tooltip as well).
https://www.deque.com/blog/creating-accessible-svgs/
https://www.smashingmagazine.com/2021/05/accessible-svg-patterns-comparison/#pattern-5-svg-role-img-title

Reply:
We really shouldn't be using the title attribute, some screen readers do not even acknowledge them at all. If we want a tooltip we can add one in but the title attribute is not encouraged anymore.

REF:
 - https://www.a11yproject.com/posts/title-attributes/
 - https://www.tpgi.com/html5-accessibility-chops-title-attribute-use-and-abuse/
 - https://yoast.com/developer-blog/dont-rely-title-attribute/

Reply:
I still don't see the change 😢

Reply:
okay, sorry idk what I was looking at. I do see the change now 😛